aplib.net-demo/Assets/Scripts/MainMenu/CanvasManager.cs

```diff
@@ -117,12 +116,11 @@ private void Start()
     private void SetAllCanvasesToInactive()
     {
         MenuCanvas.SetActive(false);
-        SettingMenuCanvas.SetActive(false);
-        SettingGameCanvas.SetActive(false);
+        SettingCanvas.SetActive(false);
         GameOverCanvas.SetActive(false);
         WinScreenCanvas.SetActive(false);
-        IsOnMenuSettings = false;
-        IsOnGameSettings = false;
+        HelpCanvas.SetActive(false);
+        IsOnSettings = false;
```

Comment:
Is this missing? I'm not sure.
```suggestion
        IsOnSettings = false;
        IsOnHelp = false;
```